opus/application/static_media/js/browse.js

```diff
@@ -20,6 +31,7 @@ var o_browse = {
 
     galleryScrollTo: 0,
     metadataSelectorDrawn: false,
+    currentSelectedMetadata: undefined,
```

Comment:
this var is only global w/in the confines of the function; it can stay where it was, maybe a diff comment would be more appropriate.